### ISSUE TITLE
unable to add 0.0.0.0/1 and 128.0.0.0/1 routes

### ISSUE THREAD

**Issue by dadoblake** [guillaumedsde/alpine-qbittorrent-openvpn#64]
Hello,
it's a few days the container fails to start, reporting errors when trying to add the default routes to 0.0.0.0/1 and 128.0.0.0/1 with the message. it used to do it in past, but rebooting the host always solved.
```
Tue Mar  2 12:49:04 2021 /usr/sbin/ip-su route add 0.0.0.0/1 via *****(vpn ip)

RTNETLINK answers: Network unreachable

Tue Mar  2 12:49:04 2021 ERROR: Linux route add command failed: external program exited with error status: 2

Tue Mar  2 12:49:04 2021 /usr/sbin/ip-su route add 128.0.0.0/1 via *****(vpn ip same as above)

RTNETLINK answers: Network unreachable

Tue Mar  2 12:49:04 2021 ERROR: Linux route add command failed: external program exited with error status: 2
```

i googled it and seems to be related to the same subnet of vpn ip and docker network, but they just start both with 172, they don't belong to the same subnet.

trying to figure it out i could have broken something within the container because it says the txt for pass is empty but of course it is not...

**Comment by PeterSchumacher** [comment]
Had trouble after upgrade to latest docker-ce. (installed from PPA - docker.com)

**TLDR; downgrade to earlier docker-ce**


Fixed it by,
- Stopping and deleting containers
- Show current and earlier versions of docker-ce ('apt-cache policy docker-ce')
- Remove installed docker-cd ('sudo apt remove --purge docker-ce docker-ce-cli docker-ce-rootless-extras')
Install earlier version
```
sudo apt install \
docker-ce=5:20.10.3~3-0~raspbian-buster \
docker-ce-cli=5:20.10.3~3-0~raspbian-buster \
docker-ce-rootless-extras=5:20.10.3~3-0~raspbian-buster
```

**Comment by marska** [comment]
On Synology first run works, but after restart container I have the same problem.
